motionless.py

```diff
@@ -247,6 +250,10 @@ def __init__(self, lat=None, lon=None, zoom=None, size_x=400, size_y=400,
         self.region = region
         self.path = []
         self.contains_addresses = False
+        if simplify_threshold_meters is None:
+            self.simplify_threshold = 0
```

Comment:
This is so the user can `specify simplify_threshold_meters=None` to *disable* path simplification. I think this makes a friendlier motionless API than having to set threshold to a value of zero, even though that's what we have to pass behind the scenes to GPolyEncode in order to disable it.